src/test/java/cs361/battleships/models/BoardTest.java

```diff
@@ -206,5 +206,70 @@ public void testSubmergedSubmarinePlacement(){
         assertTrue(board.placeShip(new Submarine("SUBMARINE", true),5,'B', false));
 
     }
+
```

Comment:
Really good tests you've written here. If you haven't already make sure you detail the confirmation tests you did for your user story on the wiki page.